samples/apps/copilot-chat-app/webapp/src/components/chat/ChatWindow.tsx

```diff
@@ -138,14 +143,18 @@ export const ChatWindow: React.FC = () => {
         setTitle(data.value);
     };
 
+    const handleSave = () => {
+        onSave().catch((e: any) => {
```

Comment:
Should we reset the `chatName` and `isEditing` if `onSave` fails? This is probably an old issue.

Reply:
not necessarily, since the user should be able to retry the operation in case of a failure and setting `isEditing` to false would close the `Popover`.